### ISSUE TITLE
Single promise for getUserAgent?

### ISSUE THREAD

**Issue by RByers** [WICG/ua-client-hints#39]
The proposal suggests this API:
```
interface mixin NavigatorUA {
  [SecureContext] Promise<NavigatorUAData> getUserAgent();
};
```

But there's discussion of potentially requiring a user-prompt for some of the more privacy-sensitive bits. Since the privacy model may be very different for, say, the "brand" and the minor version number, don't we need some ability to request some of the information but not others. I.e. perhaps each property should return it's own promise? That also makes it easier to reason about which data is being used by which sites (i.e. measure the compat risk of breaking/freezing just one field).

**Comment by yoavweiss** [comment]
I like it! Having the individual getters return Promises (or not) jives with @foolip's comment on https://github.com/WICG/ua-client-hints/issues/28#issuecomment-575101931. It will enable us to have stricter control on some bits of information but not others, without defining a separate `getLowEntropyUserAgentData` function.

**Comment by yoavweiss** [comment]
Should be closed by #48
